otelib/client.py

```diff
@@ -1,5 +1,7 @@
 """OTE Client."""
-from otelib.strategies import DataResource, Filter, Function, Mapping, Transformation
+
+from otelib.backends.python.client import OTEPythonClient
+from otelib.backends.services.client import OTEServiceClient
 
 
 class OTEClient:
```

Comment:
Perhaps this class can be the "base client class" the other backend-specific classes use as a parent?

Reply:
Yes, except it creates a circular reference. If you know the 'proper' way for a parent to return a child we can do that

Reply:
See issue https://github.com/EMMC-ASBL/otelib/issues/68